### PR TITLE
doc: connect KFP SDK to GCP endpoints

### PR DESCRIPTION
**Pull request by Bobgy**
preview: https://deploy-preview-2086--competent-brattain-de2d6d.netlify.app/docs/gke/pipelines/authentication-sdk/

Part of https://github.com/kubeflow/pipelines/issues/4317

### CODE REVIEW

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
++++
+title = "Connecting to Kubeflow Pipelines on Google Cloud using the SDK"
+description = "How to connect to different Kubeflow Pipelines installations on Google Cloud with Kubeflow Pipelines SDK?"
```
Comment (joeliedtke) [inline content/en/docs/gke/pipelines/authentication-sdk.md:3]: with Kubeflow Pipelines SDK? -> using the Kubeflow Pipelines SDK.

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+weight = 20
++++
+
+This guide describes how to connect to your Kubeflow Pipelines on Google CLoud using [the Kubeflow Pipelines SDK](/docs/pipelines/sdk/sdk-overview/).
```
Comment (joeliedtke) [inline content/en/docs/gke/pipelines/authentication-sdk.md:7]: Kubeflow Pipelines -> Kubeflow Pipelines cluster

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+## How SDK connects to Kubeflow Pipelines API
+
+Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes
+namespace you deployed it to.
+The Kubeflow Pipelines SDK can send REST api requests to this api service, but
+the SDK needs to know the hostname to connect to the API service.
+
+If the hostname can be accessed without authentication, it's very simple to
+connect to it. For example, you can use `kubectl port-forward` to access it via
```
Comment (joeliedtke) [inline content/en/docs/gke/pipelines/authentication-sdk.md:22]: Can you provide an example of how to use `kubectl port-forward`?

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+This guide describes how to connect to your Kubeflow Pipelines on Google CLoud using [the Kubeflow Pipelines SDK](/docs/pipelines/sdk/sdk-overview/).
+
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment on Google Cloud using one of the [installation options](/docs/pipelines/installation/overview/).
+* You need to [install Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+
+## How SDK connects to Kubeflow Pipelines API
+
+Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes
```
Comment (joeliedtke) [inline content/en/docs/gke/pipelines/authentication-sdk.md:16]: Suggested revision(it is ambiguous):

Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes namespace you deployed it to.

->

Kubeflow Pipelines includes an API service named `ml-pipeline-ui`. The `ml-pipeline-ui` API service is deployed in the same Kubernetes namespace you deployed Kubeflow Pipielines in.

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+When deploying Kubeflow Pipelines on Google Cloud, a public endpoint for this
+API service is auto-configured for you, but this public endpoint has security
+checks to protect your cluster from unauthorized access.
+
+The following sections introduce how to authenticate your SDK requests to connect
+to Kubeflow Pipelines via the public endpoint.
+
+## Connecting to Kubeflow Pipelines standalone or AI Platform Pipelines
+
+You can refer to [Connecting to AI Platform Pipelines using the Kubeflow Pipelines SDK](https://cloud.google.com/ai-platform/pipelines/docs/connecting-with-sdk) for
```
Comment (joeliedtke) [inline content/en/docs/gke/pipelines/authentication-sdk.md:41]: You can refer to -> Refer to

File: content/en/docs/gke/pipelines/authentication-sdk.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+## Before you begin
+
+* You need a Kubeflow Pipelines deployment on Google Cloud using one of the [installation options](/docs/pipelines/installation/overview/).
+* You need to [install Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+
+## How SDK connects to Kubeflow Pipelines API
+
+Kubeflow Pipelines has an API service named `ml-pipeline-ui` in the Kubernetes
+namespace you deployed it to.
+The Kubeflow Pipelines SDK can send REST api requests to this api service, but
```
[the remark this answers is not in the document]
Comment (Bobgy) [reply content/en/docs/gke/pipelines/authentication-sdk.md:18]: Thanks, will remember this next time!

File: content/en/docs/gke/pipelines/authentication-sdk.md (hunk 3 above)
Comment (Bobgy) [reply content/en/docs/gke/pipelines/authentication-sdk.md:22]: I put them in comments below, but it's worth making it more explicit.

**Pull request merged**